EIPS/eip-4841.md

```diff
@@ -0,0 +1,420 @@
+description: Expandable SVG images storage model for storing larger-capacity images on Ethereum.
+author: KyungEun Kim <kkekke815@gmail.com>, Soohan Park <725psh@gmail.com>, Soohan Park (@Soohan-Park)
+discussions-to: https://ethereum-magicians.org/t/eip-4841-expandable-onchain-svg-images-storage-structure/8410
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-23
+---
+
+## Abstract
+
+This standard proposal is an Expandable Onchain SVG Images Storage on Ethereum that permanently preserves images and prevents tampering, and can store larger-capacity images furthermore.
+
+It is a structure designed to store SVG images with a larger capacity by distributing SVG images in units of tags on Ethereum.
+
+The structure presented by this EIP consists of a total of three layers as shown below.
+
+![StructureDiagram.jpg](../assets/eip-4841/StructureDiagram.jpg)
+
+> **Storage Layer ─** A contract layer that stores distributed SVG images by tags.  
+> **Assemble Layer ─** A contract layer that creates SVG images by combining tags stored in the Storage Layer's contract.  
+> **Property Layer ─** A contract layer that stores the attribute values for which SVG tag to use.
+
+It is designed to flexibly store and utilize larger capacity SVG images by interacting with the above three layer-by-layer contracts each other.
+
+Also, you can configure the on-chain NFT images storage by adjusting the Assemble Layer's contract like below.
+
+- Storage with expandability by allowing additional deployment on Storage Layer's contracts
+- Storage with immutability after initial deployment
+
+Additionally, this standard proposal focuses on but is not limited to compatibility with the [EIP-721](./eip-721.md) standard.
+
+## Motivation
+
+Most NFT projects store their NFT metadata on a centralized server rather than on Ethereum. Although this method is the cheapest and easiest way to store and display the content of the NFT, there is a risk of corruption or loss of the NFT's metadata. In addition, even in the case of IPFS, tampering of contents can be prevented, but contents could be lost if there is no node storing the contents.
+
+To solve this problem, most NFT metadata is stored on Ethereum. However, it can only be expressed as a simple shape such as a circle or a rectangle, since one contract can be distributed 24 KB size for maximum.
+
+We propose this model _─ a more secure way to store NFT metadata ─_ to create and own high-quality NFT metadata.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+We would like to explain each of the three layers of the proposed model in detail.
+
+After classifying layers according to functions and roles and distributing them to different contracts, these contracts are logically connected to be used as a contract with one large storage space.
```

Comment:
As a technical article, I think you can leave out a lot of the introductory content.

```suggestion
Images stored according to this standard must be split into three layers: Storage, Assemble, and Property.
```